lib/modules/@apostrophecms/schemas/src/apos/components/AposRadioInput.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <AposInputWrapper
+    :field="field" :error="status.error"
+    :uid="uid"
+  >
+    <template #body>
+      <label
+        class="apos-choice-label" :for="getChoiceId(uid, choice.value)"
+        v-for="choice in field.choices" :key="choice.value"
+      >
+        <input
+          type="radio" class="apos-sr-only apos-input--choice apos-input--radio"
+          :value="choice.value" :name="field.name"
+          :id="getChoiceId(uid, choice.value)"
+          v-model="next" :disabled="status.disabled"
+          tabindex="1"
+        >
+        <span class="apos-input-indicator" aria-hidden="true">
+          <component
+            :is="`${next === choice.value ? 'check-bold-icon' : 'span'}`"
+            :size="8" v-if="next === choice.value"
+          />
+        </span>
+        <span class="apos-choice-label-text">
+          {{ choice.label }}
+        </span>
+      </label>
+    </template>
+  </AposInputWrapper>
+</template>
+
+<script>
+import AposInputMixin from '../mixins/AposInputMixin';
+
+export default {
+  name: 'AposRadioInput',
+  mixins: [ AposInputMixin ],
+  methods: {
+    getChoiceId(uid, value) {
+      // Convert any boolean values for this purpose.
+      if (typeof value !== 'string') {
```

Comment:
I had to handle the `null` possibility here. I thought about using the helper mixin's method, but it didn't seem like it'd work well in this case.